main.go

```diff
@@ -23,15 +25,76 @@ import (
 	"github.com/rakyll/statik/fs"
 )
 
+// Theme of longcat
+type Theme struct {
+	Head image.Image
+	Body image.Image
+	Tail image.Image
+}
+
 func loadImage(fs http.FileSystem, n string) (image.Image, error) {
 	f, err := fs.Open(n)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(fmt.Errorf("theme file does not open: %s", n))
 	}
 	defer f.Close()
 	return png.Decode(f)
 }
 
+func loadImageGlob(dir string, glob string) (image.Image, error) {
+	pattern := filepath.Join(dir, glob)
+	matches, err := filepath.Glob(pattern)
+	if err != nil {
+		return nil, err
+	}
+	if len(matches) == 0 {
+		return nil, fmt.Errorf("file does not exist: %s", pattern)
+	}
+
+	data, err := ioutil.ReadFile(matches[0])
+	if err != nil {
+		return nil, err
+	}
+
+	return png.Decode(bytes.NewReader(data))
+}
+
+func (t *Theme) loadTheme(themeName string) error {
+	fs, err := fs.New()
+	if err != nil {
+		return err
+	}
+
+	imgPath := func(s string) string {
+		return filepath.Join("/themes", themeName, s)
```

Comment:
```
return filepath.ToSlash(filepath.Join("/themes", themeName, s))
```

Reply:
Thank you for your review. Using `filepath.ToSlash` is better in this case? Why not ? 

🥴  In `go version go1.10.4 linux/amd64`: 

```golang
gore> filepath.Join("/foo", `../../buzz\mytheme`, "foo.jpg")
"/buzz\\mytheme/foo.jpg"

gore> filepath.ToSlash(filepath.Join("/foo", `\mytheme`, "foo.jpg"))
"/foo/\\mytheme/foo.jpg"

gore> filepath.ToSlash(filepath.Join("/foo", `../../buzz\mytheme`, "foo.jpg"))
"/buzz\\mytheme/foo.jpg"
```

📝 I know that `\`  is allowed in various(?) linux . 

👌 In `go version go1.12.9 windows/amd64`: 

```golang
gore> filepath.Join("/foo", `../../buzz\mytheme`, "foo.jpg")
"\\buzz\\mytheme\\foo.jpg"

gore> filepath.ToSlash(filepath.Join("/foo", `\mytheme`, "foo.jpg"))
"/foo/mytheme/foo.jpg"

gore> filepath.ToSlash(filepath.Join("/foo", `../../buzz\mytheme`, "foo.jpg"))
"/buzz/mytheme/foo.jpg"
```